Detectors/TRD/doc/diagram/diagram.xml

```diff
@@ -0,0 +1,104 @@
+<mxGraphModel dx="718" dy="687" grid="1" gridSize="10" guides="1" tooltips="1" connect="1" arrows="1" fold="1" page="1" pageScale="1" pageWidth="850" pageHeight="1100" background="none" math="0" shadow="0">
```

Comment:
so what is this documenting? Is it included somewhere?

Reply:
This is xml for a flow chart. It will be part of the TRD documentation manuscript.